text/metrics/0118-statsd-and-prometheus-exporters.md

```diff
@@ -0,0 +1,226 @@
+
+These properties will help understand how to map OpenTelemetry
+Aggregators into Prometheus and Statsd metric data.  These are the
+OpenTelemetry instruments:
+
+| Name | Synchronous | Adding or Grouping | Monotonic | Precomputed-Sum | Default Aggregator |
+| ---- | ----------- | -------- | --------- | ---- | --- |
+| Counter           | Yes | Adding   | Yes | No  | Sum |
+| UpDownCounter     | Yes | Adding   | No  | No  | Sum |
+| ValueRecorder     | Yes | Grouping | n/a | n/a | MinMaxLastSumCount |
+| SumObserver       | No  | Adding   | Yes | Yes | Sum |
+| UpDownSumObserver | No  | Adding   | No  | Yes | Sum |
+| ValueObserver     | No  | Grouping | n/a | n/a | MinMaxLastSumCount |
+
+Note that the Precomputed-Sum property places some constraints on the
+combination of Aggregator and Exporter.  To compute _Delta_
+aggregations of Precomputed-Sum instruments requires an aggregation
+that supports subtraction (which MUST include Sum and SHOULD include
+Histogram).
+
+### Prometheus
+
+This is based off of the Prometheus [Metric
+Types](https://prometheus.io/docs/concepts/metric_types).
+
+#### Prometheus instruments
+
+Prometheus Counter instruments are semantically identical to
+OpenTelemetry Counter instruments, including the Mnotonic property.
+They are exposed as a Cumulative Sum aggregation.
+
+Prometheus Gauge instruments have a number of uses, depending on
+whether they are used as an adding or as a grouping instrument.  They
+are exposed as a single data point equal to the the last value that
+was set.  Because Prometheus clients are stateful, Gauges support both
+`Set()` and `Add()` methods.  Generally, Prometheus Gauges used to
+`Add()` map into OpenTelemetry UpDownCounter instruments (maybe
+SumObserver, UpDownSumObserver), while Prometheus Gauges used to
+`Set()` map into OpenTelemetry ValueRecorder instruments (maybe
+ValueObserver).
+
+Prometheus Histogram instruments are exposed as Cumulative
+aggregations, defined by a number of bucketed counts, accumulated from
+the start of the process.  Prometheus Histogram instruments map into
+ValueRecorder instruments (maybe ValueObserver).  To configure a
+Histogram exposition in the Prometheus exporter, configure a Histogram
+Aggregator with the desired buckets.
+
+Prometheus Summary instruments are discouraged, as mentioned above,
+because they are not mergeable.  Uses of Prometheus Summary
+instruments map into OpenTelemetry ValueRecorder instruments (maybe
+ValueObserver).  To configure a Summary exposition in the Prometheus
+exporter, configure a Sketch Aggregator with the desired quantiles.
+
+#### OpenTelemetry Aggregator to Prometheus exposition
+
+The following table lists the mapping from Aggregator to Prometheus
+exposition format.  The "Typical Instruments" listed are the
+applicable OpenTelemetry instruments, for which the Prometheus mapping
+is sensible.
+
+| OpenTelemetry aggregator | Default Prometheus data type mapping | Export kind | Typical instruments | Notes |
```

Comment:
I see potentially two different people reading this table: an OpenTelemetry developer writing an exporter for prometheus, and an existing Prometheus instrumentation author looking to migrate to OpenTelemetry instrumentation.

This table seems to serve the the former well, but I'm not sure it is clear for the latter as to what they need to do. Guessing there would like a more explicit mapping between Prometheus type and OTel Instrument. I'm guessing the one-to-many mapping might be a bit overwhelming.

Is this something we can assume Prometheus instrumentation writers can work out (i.e. become familiar with synchronicity, precomputed sums, and additive/grouping)?

Is this something for a different PR?

Could this be added in a subsequent table?